release-notes/source/linux-default.inc

```diff
@@ -1,75 +1,84 @@
-imx6qdlsabreauto        linux-fslc-imx       4.1-2.0.x+git
-imx6qdlsabresd          linux-fslc-imx       4.1-2.0.x+git
-imx6qsabrelite          linux-boundary       4.1.15-2.0.0-ga+yocto
-imx6sl-warp             linux-fslc-imx       4.1-2.0.x+git
-imx6slevk               linux-fslc-imx       4.1-2.0.x+git
-imx6sllevk              linux-imx            4.9.11-1.0.0
-imx6sxsabreauto         linux-fslc-imx       4.1-2.0.x+git
-imx6sxsabresd           linux-fslc-imx       4.1-2.0.x+git
-imx6ul-pico-hobbit      linux-fslc           4.14+git
-imx6ul-pico-pi          linux-fslc           4.14+git
-imx6ulevk               linux-fslc-imx       4.1-2.0.x+git
-imx6ullevk              linux-imx            4.9.11-1.0.0
-imx7d-pico              linux-fslc           4.14+git
-imx7dsabresd            linux-fslc-imx       4.1-2.0.x+git
-imx7s-warp              linux-warp7          4.1-4.1-1.0.x-imx-warp7
-imx7ulpevk              linux-imx            4.9.11-1.0.0
-ls1012afrdm             linux-qoriq          4.9
-ls1012afrdm-32b         linux-qoriq          4.9
-ls1012ardb              linux-qoriq          4.9
-ls1012ardb-32b          linux-qoriq          4.9
-ls1021atwr              linux-qoriq          4.9
-ls1043ardb              linux-qoriq          4.9
-ls1043ardb-32b          linux-qoriq          4.9
-ls1046ardb              linux-qoriq          4.9
-ls1046ardb-32b          linux-qoriq          4.9
-ls1088ardb              linux-qoriq          4.9
-ls2080ardb              linux-qoriq          4.9
-ls2088ardb              linux-qoriq          4.9
-m28evk                  linux-fslc           4.14+git
-m53evk                  linux-denx           3.9-master
-nitrogen6sx             linux-boundary       4.1.15-2.0.0-ga+yocto
-nitrogen6x              linux-boundary       4.1.15-2.0.0-ga+yocto
-nitrogen6x-lite         linux-boundary       4.1.15-2.0.0-ga+yocto
-nitrogen7               linux-boundary       4.1.15-2.0.0-ga+yocto
-pcm052                  linux-timesys        3.13
-tx6q-10x0               linux-karo           3.16-2015-09-18
-tx6q-11x0               linux-karo           3.16-2015-09-18
-tx6s-8034               linux-karo           3.16-2015-09-18
-tx6s-8035               linux-karo           3.16-2015-09-18
-tx6u-8033               linux-karo           3.16-2015-09-18
-tx6u-80x0               linux-karo           3.16-2015-09-18
-tx6u-81x0               linux-karo           3.16-2015-09-18
-ventana                 linux-gateworks-imx  3.14-1.0.x-ga+yocto
-wandboard               linux-wandboard      4.1-2.0.x
-======================  ===================  ==========================
+======================  ===================  ====================
+        Board             Kernel Provider       Kernel Version
+======================  ===================  ====================
+apalis-imx6             linux-fslc           5.4.58+git
+cgtqmx6                 linux-congatec       ${LINUX_VERSION}+git
```

Comment:
@angolini this is likely due `linux-congatec` do not have `LINUX_VERSION` defined. Can you take a look at it or wish us to look at it?

Reply:
Yes, please do.

Reply:
https://github.com/Freescale/meta-freescale-3rdparty/pull/92
https://github.com/Freescale/meta-freescale-3rdparty/pull/93